packages/binaryninja-free.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,29 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'BinaryNinja Free'
```

Comment:
As documented in the [_Coding Conventions_ page in our Wiki](https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#toolname):
> The name of the tool being installed (usually the file name with the `.exe`)

```suggestion
  $toolName = 'binaryninja'
```